design/20190708.certificate-request-crd.md

```diff
@@ -397,8 +398,13 @@ minimal as possible in that the single goal of them is to enable its owning
 `CertificateRequest` has been observed, the general flow is as follows:
 
 - Check the group belongs to the owning `Issuer`, exit if not.
-- Check if `CertificateRequest` is in a failed state, exit if true. TODO: more
-  tightly define what a 'failed state' exactly is.
+- Check if `CertificateRequest` is in a failed state.
+  An controller may choose to add additional conditions to a failed `CertificateRequest`, but must not attempt to issue a certificate.
+  Currently failed states are:
+  - `Ready` condition with a `Failed` reason // usually set by the issuer
+  - `InvalidRequest` condition with `True` status // usually set by the issuer
+  - `Denied` condition // usually set by approver
```

Comment:
If some of these states are terminal, and others are not, this would be a great place to indicate which are which.